bundles/config/pom.xml

```diff
@@ -19,6 +19,18 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <build>
```

Comment:
This is already handled in the project pom, right? So these pom changes should not be needed.

Reply:
Yes, seems rebase on my branch/fork is off so fixing that.

Reply:
Reverted the change causing this diff.